### PR TITLE
Archer create-record bug

### PR DESCRIPTION
**Pull request by ChanochShayner**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/30019

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ChanochShayner** [comment]
The changes made in PR solved the problem for the customer, and I also added explanations to the problematic command.
There is no response from the customer for 6 days, it seems that at the moment the only problem left for the customer is sending the values correctly for creating a record.
@guykeller @adi88d FYI

**Review by adi88d — CHANGES_REQUESTED** [review]
Nice job!
please update the test `test_generate_field_cross_reference_input` to test the input when `field_val` is not a list

**Review by guykeller — APPROVED** [review]
Please change what @adi88d requested, then we're good on my side

**Comment by ChanochShayner** [comment]
@adi88d Done.

File: Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2_test.py [hunk 1]
```diff
@@ -309,9 +309,9 @@ def test_generate_field_users_groups_input():
 
 def test_generate_field_cross_reference_input():
```
Comment (adi88d) [inline Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2_test.py:310]:
```suggestion
data_test_generate_field_cross_reference_input = [
    (
        [1, 2],
        [{"ContentID": 1}, {"ContentID": 2}]
    ),
    (
         1234,
         [{"ContentID": 1234}]
    )
]

@pytest.mark.parametrize('field_value, result', data_test_generate_field_cross_reference_input)
def test_generate_field_cross_reference_input(field_value, result):
```

File: Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2_test.py [hunk 2]
```diff
@@ -309,9 +309,9 @@ def test_generate_field_users_groups_input():
...
     client = Client(BASE_URL, '', '', '', '')
-    field_key, field_value = generate_field_value(client, "", {'Type': 9}, [1, 2])
+    field_key, field_value = generate_field_value(client, "", {'Type': 9}, 1234)
```
Comment (adi88d) [inline Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2_test.py:312]: field_key, field_value = generate_field_value(client, "", {'Type': 9}, field_value)

File: Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2_test.py [hunk 3]
```diff
@@ -309,9 +309,9 @@ def test_generate_field_users_groups_input():
...
     assert field_key == 'Value'
-    assert field_value == [{"ContentID": 1}, {"ContentID": 2}]
+    assert field_value == [{"ContentID": 1234}]
```
Comment (adi88d) [inline Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2_test.py:314]:
```suggestion
    assert field_value == result
```

**Review by adi88d — CHANGES_REQUESTED** [review]
please update the test to support field value as number and as a list, like it was before

File: Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2_test.py (hunk 3 above)
Comment (ChanochShayner) [reply Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2_test.py:314]: done

File: Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2_test.py (hunk 2 above)
Comment (ChanochShayner) [reply Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2_test.py:312]: done

**Review by adi88d — APPROVED** [review]

**Pull request merged**